runtime/runtime-core/common/src/aws/smithy/kotlin/runtime/retries/RetryStrategy.kt

```diff
@@ -5,6 +5,8 @@
 
 package aws.smithy.kotlin.runtime.retries
```

Comment:
question

Shouldn't this be under `aws.smithy.kotlin.runtime.retries.strategy`?

Reply:
I didn't create a **strategy** namespace because retry strategies are the top-level entry point into the entire retry mechanism. Policies and delays are _ingredients_ to the standard retry strategy so they make sense to subcategorize but, to me, the strategies seem as close as anything to "retries themselves" so nesting them in a further namespace seems unnecessary.